pkg/cmd/pulumi/destroy.go

```diff
@@ -353,7 +353,7 @@ func newDestroyCmd() *cobra.Command {
 		"Serialize the destroy diffs, operations, and overall output as JSON")
 	cmd.PersistentFlags().IntVarP(
 		&parallel, "parallel", "p", defaultParallel,
-		"Allow P resource operations to run in parallel at once (1 for no parallelism). Defaults to unbounded.")
+		"Allow P resource operations to run in parallel at once (1 for no parallelism).")
```

Comment:
```suggestion
		"Allow P resource operations to run in parallel at once (1 for no parallelism). Defaults to 4 * number of CPUs")
```

Would this still be useful to include?  It feels easy to just assume 1 is the default otherwise, and wonder why it's getting slower with `-p 4` for example.

Reply:
Help does actually show the default, so the current code prints: 
```
-p, --parallel int Allow P resource operations to run in parallel at once (1 for no parallelism). (default 48)
```

Not that against adding that the default is "4 * cpu" as an extra description as well though but does mean something to remember to change if we we do tweak this line.

Reply:
Ah neat, I didn't realize it printed the default anyway.  I think it's better the way you have it here then!